src/main/scala/com/danielasfregola/twitter4s/http/marshalling/BodyEncoder.scala

```diff
@@ -42,6 +42,6 @@ trait BodyEncoder {
 
   // TODO - improve performance with Macros?
   private def asMap(cc: Product): Map[String, Any] =
-    cc.getClass.getDeclaredFields.map(_.getName).zip(cc.productIterator.to).toMap
+    cc.getClass.getDeclaredFields.map(_.getName).zip(cc.productIterator.toSeq).toMap
```

Comment:
`.to` can no longer infer what resulting collection type to produce - this is part of `CanBuildFrom` going away in Scala 2.13:

https://www.scala-lang.org/blog/2018/06/13/scala-213-collections.html#life-without-canbuildfrom